build.sbt

```diff
@@ -35,7 +34,6 @@ lazy val core = crossProject(JVMPlatform, JSPlatform, NativePlatform)
   .settings(
     name := "http4s-otel4s-middleware",
     libraryDependencies ++= Seq(
-      "org.typelevel" %%% "cats-core" % catsV,
```

Comment:
I'm not quite certain how/why this is no longer directly used, but it failed the build so I got rid of it

Reply:
Perhaps something no longer requires syntax, though I'd expect that would trigger a code change. :shrug:

Reply:
Maybe everything that's used is in kernel now?